Update publications.bib

duongnhatthang opened the pull request:
Changes by Thang Duong. Adding some new publications.

duongnhatthang committed:
Update publications.bib

Changes by Thang Duong. Adding some new publications.

MihaiSurdeanu commented:
Hi Duong,
@inproceedings needs "booktitle" rather "journal". Further, please use "Proceedings of..." when it's a conference.
Thanks!

duongnhatthang committed:
Changes

- Correct the @inproceedings journal to booktitle
- Updating the @article{azizi...} information

MihaiSurdeanu commented:
Thanks!

duongnhatthang commented:
Hi Prof. Mihai,
I have updated the @inproceedings and some other information.
I also want to note that, in the @proceedings, I made up the term "duong2024beyond" by myself, since we are still finishing the camera-ready version of the paper, hence, we haven't published it yet.
